Octokit.GraphQL.Core/Core/Builders/QueryBuilder.cs

```
@@ -736,6 +738,10 @@ private Expression RewritePagingConnectionExtensions(MethodCallExpression expres
                 // is visited.
                 return new AllPagesExpression((MethodCallExpression)expression.Arguments[0]);
             }
+            else if (expression.Method.GetGenericMethodDefinition() == PagingConnectionExtensions.AllPagesCustomSizeMethod)
+            {
+                return new AllPagesExpression((MethodCallExpression)expression.Arguments[0], (ConstantExpression)expression.Arguments[1]);
```

What if the value passed into `AllPages` isn't a `ConstantExpression`? We should at least throw a helpful exception rather than an `InvalidCastException` here if we don't support it. Would it be difficult to support non-constant expressions?